chain/src/main/scala/org/bitcoins/chain/config/ChainAppConfig.scala

```diff
@@ -91,12 +91,18 @@ case class ChainAppConfig(
           }
         }
       }
+      _ = startHikariLogger()
     } yield {
       logger.info(s"Applied ${numMigrations} to chain project")
     }
 
   }
 
+  override def stop(): Future[Unit] = {
+    val _ = stopHikariLogger()
+    FutureUtil.unit
```

Comment:
I see potential synchronization issues here. Should `stopHikariLogger` be run inside the `Future`?

Reply:
I think this is ok, `stopHikariLogger` and `startHikariLogger` just return a Unit so that will get fully evaluated before the trivial `FutureUtil.unit` is evaluated.